mobile/src/components/Button.tsx

```diff
@@ -0,0 +1,34 @@
+import { Pressable, Text, View } from 'react-native';
+/*
+@param name - name of the button
+@param addButtonStyle - add custom style to the button
+@param addTextStyle - add custom style to the text
+@param pilled - make the button pilled
+*/
```

Comment:
This should start with `/**` to help editors pick it up as a docstring. You should also prepend each line with `*` and indent it so the `*`s line up.

```ts
/**
 * @param name - name of the button
 * @param addButtonStyle - add custom style to the button
 * @param addTextStyle - add custom style to the text
 * @param pilled - make the button pilled
 */
```